src/terrama2/impl/DataAccessorGrADS.cpp

```diff
@@ -903,6 +934,29 @@ void terrama2::core::DataAccessorGrADS::writeVRTFile(terrama2::core::GrADSDataDe
                                                      DataSetPtr dataset) const
 {
 
+  try
+  {
+    int numBands = getNumberOfBands(dataset);
+    if(numBands > 0)
+      descriptor.tDef_->numValues_ = numBands;
```

Comment:
don't use tDef for bands, create a new field for that.
as tDef is for temporal information it can cause confusion.